Storage/tests/System/StreamWrapper/WriteTest.php

```diff
@@ -59,6 +59,15 @@ public function testFwrite()
         $this->assertFileExists($this->fileUrl);
     }
 
+    public function testTouch()
+    {
+        $this->assertFileDoesNotExist($this->fileUrl);
+
+        $this->assertTrue(touch($this->fileUrl));
```

Comment:
So, why are we not setting the metadata in this function?

Reply:
We cannot set metadata in gcs. We can only set a custom metadata. 
Since, GCS is a different file system altogether, we don't cover that functionality.